src/tests/ftest/pool/list_verbose.py

```diff
@@ -0,0 +1,332 @@
+            nvme_free=actual_nvme_free)
+
+        self.assertDictEqual(expected, actual)
+
+        # 3. Create second pool.
+        self.pool.append(self.get_pool(namespace="/run/pool_basic_2/*"))
+
+        # 4. Verify the fields for both pools.
+        self.verify_two_pools()
+
+        # 5. Exclude target 7 in rank 1 of pool 1.
+        self.pool[0].exclude(ranks=[1], tgt_idx="7")
+
+        # Sizes are reduced by 1/8.
+        scm_size_1 = self.pool[0].scm_size.value * 0.875
+        nvme_size_1 = self.pool[0].nvme_size.value * 0.875
+
+        # 6. Verify the fields for both pools with expected disabled and size.
+        self.verify_two_pools(
+            targets_disabled_1=1, scm_size_1=scm_size_1,
+            nvme_size_1=nvme_size_1)
+
+        # 7. Destroy pool 2.
+        self.pool[1].destroy()
+
+        # 8. Verify the fields for pool 1.
+        actual = self.pool[0].dmg.get_pool_list_all(verbose=True)
+        actual_scm_free, actual_nvme_free = self.get_scm_nvme_free(actual[0])
+
+        expected = self.create_expected(
+            pool=self.pool[0], scm_free=actual_scm_free,
+            nvme_free=actual_nvme_free, targets_disabled=1, scm_size=scm_size_1,
+            nvme_size=nvme_size_1)
+
+        self.assertDictEqual(expected, actual[0])
+
+        # 9. Verify that there's only one pool in the list.
+        self.assertEqual(len(actual), 1)
+
+        # 10. Destroy pool 1.
+        self.pool[0].destroy()
+
+        # 11. Verify that the list is empty.
+        actual = self.pool[0].dmg.get_pool_list_all(verbose=True)
+        self.assertEqual(len(actual), 0)
+
+    def verify_used_imbalance(self, storage):
+        """Verification steps for test_used_imbalance.
+
+        Args:
+            storage (str): NVME or SCM.
+
+        Returns:
+            list: Errors.
+
+        """
+        # 1. Create a pool with 80GB.
+        if storage == "NVME":
+            self.add_pool(namespace="/run/pool_both/*")
+        else:
+            self.add_pool(namespace="/run/pool_scm_only/*")
+
+        # 2. Store free.
+        before = self.pool.dmg.get_pool_list_all(verbose=True)[0]
+        free_before = -1
+        for usage in before["usage"]:
+            if usage["tier_name"] == storage:
+                free_before = usage["free"]
+
+        # 3, 4. Create an S1 container and run 1GB IOR with S1.
+        cmd_result = self.run_ior_with_pool()
+        metrics = IorCommand.get_ior_metrics(cmd_result)
+        ior_write_size = int(metrics[0][22])
+        self.log.info("IOR metrics = %s", metrics)
+        self.log.info("ior_write_size = %d", ior_write_size)
+
+        # Obtain the new free and imblanace.
+        after = self.pool.dmg.get_pool_list_all(verbose=True)[0]
+        free_after = 0
+        imbalance = 0
+        for usage in after["usage"]:
+            if usage["tier_name"] == storage:
+                free_after = usage["free"]
+                imbalance = usage["imbalance"]
+                break
```

Comment:
Optionally we could have `self.get_scm_nvme_free()` include the imbalance data in a dictionary of information it returns.